hash/nes.xml

```diff
@@ -56813,7 +56811,7 @@ preliminary proto for the PAL version, still running on NTSC systems) or the gfx
 	<software name="chuliuxx" supported="no">
 		<description>Chu Liu Xiang Xin Zhuan (Chi)</description>
```

Comment:
This should be "Chǔliúxiāng Xīn Chuán"

Reply:
Chu Liuxiang is a (fictional) person's name: https://en.wikipedia.org/wiki/Chu_Liuxiang. See earlier comment about "xinzhuan".